docs/how-to/address-ipfs-on-web.md

```diff
@@ -30,6 +30,24 @@ HTTP gateways have worked well since 2015, but they come with a significant set
 
 Tools and browser extensions should detect IPFS content paths and resolve them directly over IPFS protocol. They should use HTTP gateway only as a fallback when no native implementation is available in order to ensure a smooth, backward-compatible transition.
 
+::: tip
+
+Use relative or absolute URLs that include content-addressed paths. This will take advantage of content-addressing today while ensuring backward-compatibility with the old web.
```

Comment:
```suggestion
Use relative or absolute URLs that include content-addressed paths. This will take advantage of content addressing today while ensuring backward compatibility with the legacy web.
```